modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/tests/OpenAPIProfile.testcase

```diff
@@ -8,7 +8,7 @@ definition {
 	setUp {
 		TestCase.setUpPortalInstance();
 
-		User.firstLoginPG();
+		SignIn.signInTestSetup();
```

Comment:
We updated to this is because when we set `portal.proxy.path`, we already config Apache to url(:88/{path}) to listen port 88, and the current port we're getting and removing a port with 4 digits in removePortNumber().